packages/typescript-resolver-files/src/validatePresetConfig/validatePresetConfig.ts

```diff
@@ -460,3 +471,24 @@ const validateAddOption = (
   // TODO: a bit hacky here but what's a good way to coerce this type? 🤔
   return add as ParsedPresetConfig['add'];
 };
+
+const parseMergeSchema = (
+  mergeSchema: RawPresetConfig['mergeSchema']
+): ParsedPresetConfig['mergeSchema'] => {
+  if (!mergeSchema) {
+    return false;
+  }
+
+  if (mergeSchema === true) {
+    return { path: 'schema.generated.graphql', config: {} };
```

Comment:
The default `schema.generated.graphql` can be picked up by the majority of schema setup, I think. 
Because people may use `**/*.graphql` which will pick up both the generated version and the unmerged version 🤔

Reply:
Changed the default to `schema.generated.gql` to avoid this issue